src/com/service/PersonService.java

```diff
@@ -0,0 +1,68 @@
+package com.service;
+
+import com.person.Person;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class PersonService implements IService{
+
+    private List<Person> allPersons;
+
+    public PersonService() {
+        this.allPersons = new ArrayList<>();
+    }
+
+
+    public Person isPersonCreated(UUID uuid) {
```

Comment:
When you have methods like is..SomethingSomething its expected that they return true or false. So if this implementation stays you should name it findPersonByUUID or something like it